src/annotator/config/index.js

```diff
@@ -15,26 +12,17 @@ function configFrom(window_) {
 
   var config = {
     app: settings.app,
-
-    // Extract the default annotation ID or query from the URL.
-    //
-    // The Chrome extension or proxy may already have provided this config via
-    // a tag injected into the DOM, which avoids the problem where the page's
-    // JS rewrites the URL before Hypothesis loads.
-    //
-    // In environments where the config has not been injected into the DOM,
-    // we try to retrieve it from the URL here.
     query: settings.query,
     annotations: settings.annotations,
+    openLoginForm: settings.hostPageSetting('openLoginForm'),
+    openSidebar: settings.hostPageSetting('openSidebar'),
```

Comment:
I've just noticed that we _do_ set these settings specifically in order for the extension to pick them up in the `/welcome` route on the service. See `help.html.jinja2` in the h repo.